server/controllers/users.js

```diff
@@ -46,11 +46,11 @@ module.exports.signup = async (req, res) => {
 
     if (existingUser) {
       console.log(existingUser)
-      return res.status(400).json({ message: 'User already exists.' })
+      return res.status(StatusCodes.BAD_REQUEST).send('User already exists! Try new email address.')
     }
 
     if (password !== confirmPassword)
-      return res.status(400).json({ message: "Password don't match." })
+    return res.status(StatusCodes.BAD_REQUEST).send("Password don't match")
```

Comment:
I missed it the first time, but I think it should be "Password doesn't match." Otherwise, I believe everything else looks okay.